packages/server/src/restaurants/database/DishesSchema.ts

```diff
@@ -0,0 +1,185 @@
+import { Prop, raw, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Exclude, Expose } from 'class-transformer';
+import { ObjectId } from 'mongodb';
+import mongoose, { Document } from 'mongoose';
+
+type DishDocument = Dish & Document<ObjectId>;
+
+enum DishTags {
+  Vegan = 'wegańska',
+  Vegetarian = 'wegetariańska',
+  GlutenFree = 'gluten free',
+  Spicy = 'ostre',
+  VerySpicy = 'bardzo ostre',
```

Comment:
Nie jestem w 100% przekonany do tego, chyba lepszym rozwiązaniem byłoby trzymanie angielskich nazw, a tłumaczenie powierzyć klientowi? Musiałby się ktoś jeszcze wypowiedzieć, bo wygląda mi to na code smell, ale nie potrafię dobrze uargumentować dlaczego 😆

Reply:
Ja mam podobny problem w `RestaurantSchema`, też mam przetłumaczone ale dziwnie mi to wygląda,  do tego zastanawiam się czy by w ogóle tych enumów gdzieś nie przenieść żeby były w jednym miejscu

Reply:
Od Marty zgapiłem z tego drafta xD więc no można to przemyśleć bo jest trochę dziko xd

Reply:
@htk4 chyba najlepiej jak ekspert się wypowie na ten temat 🧐

Reply:
No trochę dziko, ale w naszym przypadku może wystarczająco :thinking: ?
Ustandaryzowane nazwy przydałyby się gdybyśmy:
- współdzielili dostęp do bazy z innymi systemami (my mamy tylko jeden serwer, więc i tak przy query do bazy danych będziemy używać nazw z enum. Dzięki temu możemy uniknąć dodatkowego mapowania nazw czy to po stronie klienta czy serwera)
- musieli wspierać więcej niż jeden język (wtedy w zależności, czy musimy wspierać backend czy frontend, tam następowałoby mapowanie pomiędzy naszymi tokenami a językiem oczekiwanym przez użytkownika)
- pracowali w międzynarodowym zespole